docs/introduction.md

```diff
@@ -2,9 +2,23 @@
 sidebar_position: 1
 ---
 # Why Unv?
-
-> I had learned many programming languages. This is my attempt to create a unique one for any platform.
->
-> _Kavindu Santhusa (Creater)_
-
-Currently Unv is in development.
+--

```

Comment:
```suggestion

```
`--` is wrong. you should use `---` for horizontal rules.
It's unnecessary here. headings have enough bottom margin.